taskfiles/lint.yaml

```diff
@@ -78,14 +78,16 @@ tasks:
       - task: "py"
         vars:
           BLACK_FLAGS: "--check --diff"
-          RUFF_FLAGS: ""
+          RUFF_CHECK_FLAGS: ""
+          RUFF_FORMAT_FLAGS: "--check --diff"
 
   fix-py:
     cmds:
       - task: "py"
         vars:
           BLACK_FLAGS: ""
-          RUFF_FLAGS: "--fix"
+          RUFF_CHECK_FLAGS: "--exit-non-zero-on-fix --fix"
+          RUFF_FORMAT_FLAGS: "--exit-non-zero-on-fix"
```

Comment:
Why do we want `--exit-non-zero-on-fix`?

Reply:
yes we should just fix instead of making the task rerun after a successful fix. going to remove these 2